tests/buildid.c

```diff
@@ -0,0 +1,17 @@
+// Checks that we generate BuildID
+//
+// REQUIRES: clang, lld
+// RUN: %clang -fuse-ld=lld -flto -O2 %S/Inputs/foo.c %s -o %t
+// RUN: %t
+// RUN: file %t|grep -q BuildID
```

Comment:
The output of `file` is not reliable across systems (on my system, your test fails). What's the goal of this test? Check that a GNU_BUILD_ID note is set? If so, maybe rely on the output of `readelf -a` (or better, of `llvm-objdump`) ?

Reply:
Correct, I want to test buildid, it caused me some pain when I moved to stage2  (I still have nightmare ;)

Reply:
fixed